README.md

```diff
@@ -0,0 +1,50 @@
+# PHP Password Checker
+
+Passwords must be at least 10 characters in length and not be commonly used - there's no means to override this.
+
+Numeric *looking* passwords are rejected, to weed out obvious memorable dates and phone numbers.
+
+All password checks are **case insensitive**.
+
+```
+$checker = new PasswordChecker;
+$checker->validate('abc123'); // throws PasswordException (too short)
+$checker->validate('password123'); // throws PasswordException (too common)
+$checker->validate('123-456-7890'); // throws PasswordException (too numeric)
+$checker->validate('31/12/1999'); // throws PasswordException (too numeric)
+$checker->validate('we love php'); // returns true
+```
+
+That's it. Though you can add further (optional, but recommended) checks and restrictions...
```

Comment:
I don't think an ellipsis is correct here.

Reply:
@WillJW Cheers, have removed.